packages/google-oauth/package.js

```diff
@@ -0,0 +1,17 @@
+Package.describe({
+  summary: "Google OAuth flow",
+  version: "1.2.0"
+});
+
+Package.onUse(function(api) {
+  api.use('oauth2', ['client', 'server']);
+  api.use('oauth', ['client', 'server']);
+  api.use('http', ['server']);
+  api.use(['underscore', 'service-configuration'], ['client', 'server']);
+  api.use(['random', 'templating@1.2.13'], 'client');
+
+  api.export('Google');
```

Comment:
Nit pick, but this should be after `api.addFiles` just to match the [Facebook package](https://github.com/meteor/meteor/blob/a4dad54a097f3630a1e583865044be5823e13621/packages/facebook-oauth/package.js)